include/umf/pools/pool_jemalloc.h

```diff
@@ -34,15 +34,6 @@ umfJemallocPoolParamsCreate(umf_jemalloc_pool_params_handle_t *hParams);
 umf_result_t
 umfJemallocPoolParamsDestroy(umf_jemalloc_pool_params_handle_t hParams);
 
-/// @brief  Set if \p umfMemoryProviderFree() should never be called.
```

Comment:
Hmm, now we have no params for the jemalloc pool.
Do we need `umfJemallocPoolParamsCreate` and  `umfJemallocPoolParamsDestroy` functions at all?

Reply:
We do not need them now. I have left them for now in case they will be needed in the future. 
I am not sure if they should be removed or not, but I can do it now, of course.
@vinser52 @bratpiorka what do you think?

Reply:
Yeah. It might confuse users that such API exists but is useless.
I vote to remove it. We can easily add it in the future.

Reply:
agree to remove it

Reply:
Removed.
Done.